packages/cli/src/models/network/NetworkBaseController.js

```diff
@@ -86,6 +86,7 @@ export default class NetworkBaseController {
     if (this._newVersionRequired()) {
       log.info(`Current version ${this.currentVersion}`);
       log.info(`Creating new version ${this.packageVersion}`);
+      this.networkFile.frozen = false;
```

Comment:
we can remove the overload from `NetworkLibController` then, I know that we are not removing lib code in this PR tho